boards/native/Makefile.dep

```diff
@@ -14,6 +16,7 @@ ifneq (,$(filter periph_can,$(FEATURES_USED)))
 endif
 
 ifneq (,$(filter socket_zep,$(USEMODULE)))
+  DEFAULT_MODULE += socket_zep_hello
```

Comment:
Oh... it is a default module... why?

Reply:
There is no way to find out if an application should be used with a dispatcher. e.g. we could only add it to `examples/gnrc_border_router`, but then if other apps would use it, it would just mysteriously not work.

Reply:
Then this should be documented. It is a proprietary extension for our ZEP dispatcher. IMHO, this should be opt-in rather than opt-out.

Reply:
I added a note to the documentation.

Reply:
Ermm... I meant that the opt-in mechanism should be documented in the README of `zep_dispatch` (and / or the `gnrc_border_router` example), not that the opt-out mechanism should be kept and stay in the documentation...